src/tracing/instrumentation/httpServer.js

```diff
@@ -124,3 +129,19 @@ function getExistingTraceId(req, fallback) {
 
   return traceId;
 }
+
+
+function filterParams(queryString) {
+  if (!queryString ||
+      queryString === '' ||
+      !(typeof queryString === 'string')) {
+    return queryString;
+  }
+  return queryString.split('&').filter(function(param) {
+    var key = param.split('=')[0];
+    if (key) {
+      return !agentOpts.secretsMatcher(key);
```

Comment:
Maybe rename `secretsMatcher` to something that hints at the response? Wasn't clear to me when reading this whether true of false means to hide the data. So possibly something like `isAllowedKey` or something?

Reply:
You still got it backwards, which might prove your point, I guess. It is called `isSecret` in Java land. If it returns true, the key must be scrubbed. (Note the `!` in line 143).

Reply:
Renamed it to `isSecret`.